src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/IdamServiceImpl.java

```diff
@@ -105,7 +109,7 @@ public AttributeResponse updateUserDetails(UpdateUserDetails updateUserDetails,
             Response response = idamClient.updateUserDetails(updateUserDetails, userId);
             responseEntity = JsonFeignResponseHelper.toResponseEntity(response, getResponseMapperClass(response, null));
         } catch (FeignException ex) {
-            log.error("SIDAM call failed:", ex);
+            log.error("{}::, {}, {}", loggingComponentName, "SIDAM call failed:", ex);
```

Comment:
comma not required

Reply:
Addressed in this commit: https://github.com/hmcts/rd-user-profile-api/pull/395/commits/308f72cc18bb9e89d2c086c252d57bf1a17ebd6a